### PR TITLE
[add-config-param-table] add config param table in README.md

### PR DESCRIPTION
**Pull request by aliartiza75**
adding the config param table in the repo's README.md

### CODE REVIEW

**Comment by aliartiza75** [comment]
@naseemkullah kindly have a look at this PR, thank you.

File: charts/influxdb/README.md [hunk 1]
```diff
@@ -46,6 +46,61 @@ The command removes all the Kubernetes components associated with the chart and
 
 The default configuration values for this chart are listed in `values.yaml`.
```
Comment (naseemkullah) [inline charts/influxdb/README.md]:
```suggestion
```

File: charts/influxdb/README.md [hunk 2]
```diff
@@ -46,6 +46,61 @@ The command removes all the Kubernetes components associated with the chart and
...
 
+#### General
```
Comment (naseemkullah) [inline charts/influxdb/README.md]:
```suggestion
## Configuration

The following table lists the configurable parameters of the InfluxDB chart and their default values.
```

**Comment by aliartiza75** [comment]
hi @rawkode, thank you for reviewing and approving this PR.

File: charts/influxdb/README.md [hunk 3]
```diff
@@ -46,6 +46,61 @@ The command removes all the Kubernetes components associated with the chart and
 
 The default configuration values for this chart are listed in `values.yaml`.
 
+#### General 
+
+| Parameter | Description | Default |
+|---|---|---|
+| image.repository | Image repository url | influxdb |
+| image.tag | Image tag | 1.7.6-alpine |
```
Comment (naseemkullah) [inline charts/influxdb/README.md]:
```suggestion
| image.tag | Image tag | 1.7.10-alpine |
```

File: charts/influxdb/README.md [hunk 4]
```diff
@@ -46,6 +46,61 @@ The command removes all the Kubernetes components associated with the chart and
...
+| config.udp | [Details](https://docs.influxdata.com/influxdb/v1.7/administration/config/#udp) | {} |
+| config.continous_queries | [Details](https://docs.influxdata.com/influxdb/v1.7/administration/config/#continuous-queries) | {} |
+| config.tls | [Details](https://docs.influxdata.com/influxdb/v1.7/administration/config/#tls) | {} |
+| initScripts.enabled | Boolean flag to enable and disable initscripts. If the container finds any files with the extensions .sh or .iql inside of the /docker-entrypoint-initdb.d folder, it will execute them. The order they are executed in is determined by the shell. This is usually alphabetical order. | false |
+| initScripts.scripts | Init scripts | {} |
+| backup.enabled | Boolean flag to enable and disable backups. Currently, it backups the data on `azure` and `gcs`. | false |
+| backup.schedule | Cron time | `0 0 * * *`. It means create a backup everyday at `00:00`. |
+| backup.annotations | Annotations for backup | {} |
+
+
```
Comment (naseemkullah) [inline charts/influxdb/README.md]:
```suggestion
```

**Comment by naseemkullah** [comment]
Thanks @aliartiza75! I added some suggestions.

**Comment by aliartiza75** [comment]
@naseemkullah thank you for reviewing the PR, i have resolved your comments, Kindly re-review it. Thank you.

**Comment by naseemkullah** [comment]
Thanks @aliartiza75 for this much needed values table! It seems like enterprise values are missing but those can be added in a subsequent PR.

**Pull request merged**